AntWars/Helper/CostCalculator.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AntWars.Board.Ants;
+
+namespace AntWars.Helper
+{
+    class CostCalculator
+    {
+        /// <summary>
+        /// Berechnet die Kosten anhand der übergebenen Ameise.
+        /// </summary>
+        /// <param name="ant">Die Ameise</param>
+        public static int calculateCost(Ant ant)
+        {
+            if (ant.isCarry())
+            {
+                return calculateCostCarry(ant.ViewRange, ant.MoveRange, ant.MaxInventory);
+            }
+            if (ant.isScout())
+            {
+                return calculateCostScout(ant.ViewRange, ant.MoveRange, ant.MaxInventory);
+            }
+            throw new ArgumentException(Messages.ERROR_INVALID_ANTTYPE);
```

Comment:
exception nachrichten haben wir bisjetzt immer auf englisch und ohne constanten in der message.cs gemacht, das sollte einheitlich bleiben.